### PR TITLE
WINC-1310: Remove redundant AWS upgrade job and use Windows Server 2019 in Azure operator test

### PR DESCRIPTION
**Pull request by saifshaikh48**
- Removes the old upgrade test, as it has been replaced by the newer upgrade test job for a while now.
  - `azure-e2e-upgrade` runs on all WMCO branches except release-4.14 which currently runs `vsphere-e2e-upgrade`
- Change `azure-e2e-operator` to use Windows Server 2019

### CODE REVIEW

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.17.yaml [hunk 1]
```diff
@@ -213,44 +213,6 @@ tests:
...
-          memory: 400Mi
-    - as: e2e-aws-upgrade
-      cli: latest
-      commands: |-
-        #!/bin/bash
-        set -euo pipefail
-        cluster_profile=/var/run/secrets/ci.openshift.io/cluster-profile
-        export AWS_SHARED_CREDENTIALS_FILE=${cluster_profile}/.awscred
-        export KUBE_SSH_KEY_PATH=${cluster_profile}/ssh-privatekey
-        export WINDOWS_SERVER_VERSION="2019"
```
Comment (jrvaldes) [inline ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.17.yaml:245]: looks like there won't be any Windows Server 2019 coverage after this merges.

**Review by jrvaldes — APPROVED** [review]
/lgtm

**Comment by saifshaikh48** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-azure-e2e-operator pull-ci-openshift-windows-machine-config-operator-release-4.16-azure-e2e-operator pull-ci-openshift-windows-machine-config-operator-release-4.14-azure-e2e-operator

**Review by jrvaldes — APPROVED** [review]
/lgtm

**Comment by jrvaldes** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-azure-e2e-operator

To gather more data points.

**Comment by saifshaikh48** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-release-4.12-azure-e2e-operator pull-ci-openshift-windows-machine-config-operator-release-4.13-azure-e2e-operator pull-ci-openshift-windows-machine-config-operator-release-4.15-azure-e2e-operator pull-ci-openshift-windows-machine-config-operator-release-4.16-azure-e2e-operator

**Comment by saifshaikh48** [comment]
/rehearsals-ack

saw some issues with file transfer on 2019 nodes but a retest proved them to be flakes

**Pull request merged**